SAMv1.tex

```diff
@@ -362,23 +364,28 @@ \subsection{The alignment section: mandatory fields}\label{sec:alnrecord}
 \item {\sf CIGAR}: CIGAR string. The CIGAR operations are given in the
   following table (set `*' if unavailable):
   \begin{center}\small
-  \begin{tabular}{ccl}
+% \begin{tabular}{ccp{2cm}p{2cm}l}
+  \begin{tabular}{ccccl}
   \hline
-  Op & BAM & Description\\
+%  Op & BAM & Consumes \newline reference & Consumes \newline sequence & Description\\
+  Op & BAM & \makecell{Consumes \\ query} & \makecell{Consumes \\ reference} & Description\\
   \hline
-  {\tt M} & 0 & alignment match (can be a sequence match or mismatch)\\
-  {\tt I} & 1 & insertion to the reference \\
-  {\tt D} & 2 & deletion from the reference \\
-  {\tt N} & 3 & skipped region from the reference \\
-  {\tt S} & 4 & soft clipping (clipped sequences present in {\sf SEQ})\\
-  {\tt H} & 5 & hard clipping (clipped sequences NOT present in {\sf SEQ})\\
-  {\tt P} & 6 & padding (silent deletion from padded reference)\\
-  {\tt =} & 7 & sequence match \\
-  {\tt X} & 8 & sequence mismatch \\
+  {\tt M} & 0 & yes & yes & alignment match (can be a sequence match or mismatch)\\
+  {\tt I} & 1 & yes & no  & insertion to the reference \\
+  {\tt D} & 2 & no  & yes & deletion from the reference \\
+  {\tt N} & 3 & no  & yes & skipped region from the reference \\
+  {\tt S} & 4 & yes & no  & soft clipping (clipped sequences present in {\sf SEQ})\\
+  {\tt H} & 5 & no  & no  & hard clipping (clipped sequences NOT present in {\sf SEQ})\\
+  {\tt P} & 6 & no  & no  & padding (silent deletion from padded reference)\\
+  {\tt =} & 7 & yes & yes & sequence match \\
+  {\tt X} & 8 & yes & yes & sequence mismatch \\
   \hline
   \end{tabular}
   \end{center}
   \begin{itemize}
+  \item ``Consumes query'' and ``Consumes reference'' indicates
+    whether the CIGAR operation causes the alignment to step along the
+    query sequence and / or the reference sequence.
```

Comment:
I'd use "indicate" (plural) above and "sequence and the reference sequence, respectively." here, otherwise, 👍